modules/pre1api.js

```diff
@@ -124,7 +124,7 @@ pbjs.requestBids.addHook((config, next = config) => {
   } else {
     logWarn(`${MODULE_NAME} module: concurrency has been disabled and "$$PREBID_GLOBAL$$.requestBids" call was queued`);
   }
-}, 100);
+}, 5);
```

Comment:
The number (also relates to the other comment below) acts as a priority level for the hooked functions (when there are multiple hooked functions on the same hook).

The `pbjs.requestBids` function has 3 hooked functions from 3 different pbjs modules: pre1api, PubCommonId and consentManagement.  So lowering this priority level ensures that the `pre1api` module's hooked function will (generally) go last in the sequence if all these modules were enabled together.

Having the `pre1api` module go last (specifically after `consentManagement`) is related to the need for the `pre1api` to execute right before the `pbjs.requestBids` so it knows which auction is currently active.  The `consentManagement` module uses/waits on callbacks (to retrieve information from the CMP), and this buffering of the auction process could cause issues for the `pre1api` knowing which auction is active if there was a time gap because it ran first.  So the lower priority helps avoid this scenario.

The 5 specifically is lower than the default priority that's set for any hooked function (which is 10), but still higher than the base function `pbjs.requestBids` (which is 0).  So it should generally always the be last hook to run before the `pbjs.requestBids` would execute.